components/downstate/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@spectrum-css/downstate",
+  "version": "1.0.0-alpha.0",
+  "description": "The Spectrum CSS downstate utility class",
+  "license": "Apache-2.0",
+  "author": "Adobe",
+  "homepage": "https://opensource.adobe.com/spectrum-css/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/spectrum-css.git",
+    "directory": "components/downstate"
```

Comment:
Is there a reason to keep this down state "concept" as a part of the component directory?

Reply:
Nope! After this PR was opened we got some input from design, so this will change to look more like the [corner rounding PR](https://github.com/adobe/spectrum-css/pull/2559) (which, disclaimer, is also a work in progress like this one)

Reply:
I've pushed up a WIP commit that starts making the changes needed for this.